src/Model/GroundWaterTransport/gwt-src.f90

```diff
@@ -79,8 +88,37 @@ subroutine src_create(packobj, id, ibcnum, inunit, iout, namemodel, pakname, &
     !
     ! -- Store the appropriate label based on the dependent variable
     srcobj%depvartype = depvartype
+
+    srcobj%fmi => fmi
+
   end subroutine src_create
 
+  !> @brief Set additional options specific to the GwtSrcType
+  !!
+  !! This routine overrides BndType%bnd_options
+  !<
+  subroutine src_options(this, option, found)
+    ! -- dummy
+    class(GwtSrcType), intent(inout) :: this
+    character(len=*), intent(inout) :: option
+    logical, intent(inout) :: found
+    ! -- local
+    ! -- formats
+
+    found = .true.
+    select case (option)
+    case ('HIGHEST_SATURATED')
+      this%highest_sat = .TRUE.
+      write (this%iout, '(4x,a)') &
+        'SOURCES WILL BE APPLIED TO THE HIGHEST CELL AT OR BELOW THE SPECIFIED &
+        &CELLID WITH A NON-ZERO CELL SATURATION'
```

Comment:
```suggestion
        &cellid with a non-zero saturation.'
```

Reply:
@christianlangevin added a test with wet dry. Saturation is calculated for a dry cell. 

https://github.com/MODFLOW-ORG/modflow6/blob/cad280f647756be45d5336f60c4291638547f326/src/Model/GroundWaterFlow/gwf-npf.f90#L467